indexing-solr/src/test/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilterTest.java

```diff
@@ -363,4 +362,18 @@ public void fileTitle() {
 
         assertEquals("uuid: 1234", idb.getTitle());
     }
+
+    @Test
+    public void testDateCreatedYearBxc3941() throws Exception {
```

Comment:
is the BXC3941 the ticket number? that might be confusing in the future

Reply:
Yes, its the ticket number. I've seen other projects list the ticket that describes the bug being tested for. I could put it as a comment rather than in the name of the method if that's clearer

Reply:
ohh i hadn't seen that convention before. interesting! i guess it's fine with me to leave it